[dv-processing] Build failure on arm64-osx

Package: dv-processing[core,tools]:arm64-osx -> 1.7.9#1

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 15.0.0.15000309
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 3d72d8c93 2024-07-10 (13 hours ago)

**To Reproduce**

`vcpkg x-set-installed --enforce-port-checks --allow-unsupported dv-processing[core,tools]`
**Failure logs**

```
-- Using cached dv-dv-processing-9cd21eede0c38e079e462cdce8434fcbe2a4d037.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/dv-dv-processing-9cd21eede0c38e079e462cdce8434fcbe2a4d037.tar.gz
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean
-- Using cached dv-cmakemod-d107c76b73a49a16c3ac733749152037406a515e.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/dv-cmakemod-d107c76b73a49a16c3ac733749152037406a515e.tar.gz
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/37406a515e-9278402073.clean
-- Found external ninja('1.12.1').
-- Configuring arm64-osx-rel
-- Building arm64-osx-rel
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.29.2-osx/cmake-3.29.2-macos-universal/CMake.app/Contents/bin/cmake --build . --config Release --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/arm64-osx-rel
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/install-arm64-osx-rel-out.log

Call Stack (most recent call first):
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/dv-processing/portfile.cmake:45 (vcpkg_cmake_install)
  scripts/ports.cmake:192 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/install-arm64-osx-rel-out.log</summary>

```
Change Dir: '/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/arm64-osx-rel'

Run Build Command(s): /opt/homebrew/bin/ninja -v -v -j11 install
[1/9] /opt/homebrew/bin/ccache ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DBOOST_NOWIDE_NO_LIB -DBOOST_STACKTRACE_GNU_SOURCE_NOT_REQUIRED -DCLI11_COMPILE -DHAVE_PTHREADS=1 -DOS_MACOS=1 -DOS_UNIX=1 -D_DARWIN_C_SOURCE=1 -D_DEFAULT_SOURCE=1 -D_FILE_OFFSET_BITS=64 -D_POSIX_C_SOURCE=200809 -D_XOPEN_SOURCE=700 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/eigen3 -fPIC -O3 -DNDEBUG -std=c++20 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.5.sdk -MD -MT utilities/list-devices/CMakeFiles/dv-list-devices.dir/list-devices.cpp.o -MF utilities/list-devices/CMakeFiles/dv-list-devices.dir/list-devices.cpp.o.d -o utilities/list-devices/CMakeFiles/dv-list-devices.dir/list-devices.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/utilities/list-devices/list-devices.cpp
FAILED: utilities/list-devices/CMakeFiles/dv-list-devices.dir/list-devices.cpp.o 
/opt/homebrew/bin/ccache ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DBOOST_NOWIDE_NO_LIB -DBOOST_STACKTRACE_GNU_SOURCE_NOT_REQUIRED -DCLI11_COMPILE -DHAVE_PTHREADS=1 -DOS_MACOS=1 -DOS_UNIX=1 -D_DARWIN_C_SOURCE=1 -D_DEFAULT_SOURCE=1 -D_FILE_OFFSET_BITS=64 -D_POSIX_C_SOURCE=200809 -D_XOPEN_SOURCE=700 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/eigen3 -fPIC -O3 -DNDEBUG -std=c++20 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.5.sdk -MD -MT utilities/list-devices/CMakeFiles/dv-list-devices.dir/list-devices.cpp.o -MF utilities/list-devices/CMakeFiles/dv-list-devices.dir/list-devices.cpp.o.d -o utilities/list-devices/CMakeFiles/dv-list-devices.dir/list-devices.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/utilities/list-devices/list-devices.cpp
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/utilities/list-devices/list-devices.cpp:1:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/core/utils.hpp:4:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/core/../external/compare_compat.hpp:25:16: error: no template named '__is_cpp17_random_access_iterator'
        if constexpr (__is_cpp17_random_access_iterator<_InputIterator1>::value
                      ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/core/../external/compare_compat.hpp:26:10: error: no template named '__is_cpp17_random_access_iterator'
                                  && __is_cpp17_random_access_iterator<_InputIterator2>::value) {
                                     ^
2 errors generated.
[2/9] /opt/homebrew/bin/ccache ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DBOOST_NOWIDE_NO_LIB -DBOOST_STACKTRACE_GNU_SOURCE_NOT_REQUIRED -DCLI11_COMPILE -DHAVE_PTHREADS=1 -DOS_MACOS=1 -DOS_UNIX=1 -D_DARWIN_C_SOURCE=1 -D_DEFAULT_SOURCE=1 -D_FILE_OFFSET_BITS=64 -D_POSIX_C_SOURCE=200809 -D_XOPEN_SOURCE=700 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/eigen3 -fPIC -O3 -DNDEBUG -std=c++20 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.5.sdk -MD -MT utilities/filestat/CMakeFiles/dv-filestat.dir/filestat.cpp.o -MF utilities/filestat/CMakeFiles/dv-filestat.dir/filestat.cpp.o.d -o utilities/filestat/CMakeFiles/dv-filestat.dir/filestat.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/utilities/filestat/filestat.cpp
FAILED: utilities/filestat/CMakeFiles/dv-filestat.dir/filestat.cpp.o 
/opt/homebrew/bin/ccache ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DBOOST_NOWIDE_NO_LIB -DBOOST_STACKTRACE_GNU_SOURCE_NOT_REQUIRED -DCLI11_COMPILE -DHAVE_PTHREADS=1 -DOS_MACOS=1 -DOS_UNIX=1 -D_DARWIN_C_SOURCE=1 -D_DEFAULT_SOURCE=1 -D_FILE_OFFSET_BITS=64 -D_POSIX_C_SOURCE=200809 -D_XOPEN_SOURCE=700 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/eigen3 -fPIC -O3 -DNDEBUG -std=c++20 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.5.sdk -MD -MT utilities/filestat/CMakeFiles/dv-filestat.dir/filestat.cpp.o -MF utilities/filestat/CMakeFiles/dv-filestat.dir/filestat.cpp.o.d -o utilities/filestat/CMakeFiles/dv-filestat.dir/filestat.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/utilities/filestat/filestat.cpp
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/utilities/filestat/filestat.cpp:4:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/io/read_only_file.hpp:3:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/io/reader.hpp:3:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/io/compression/decompression_support.hpp:5:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/io/compression/../../core/utils.hpp:4:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/io/compression/../../core/../external/compare_compat.hpp:25:16: error: no template named '__is_cpp17_random_access_iterator'
        if constexpr (__is_cpp17_random_access_iterator<_InputIterator1>::value
                      ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/io/compression/../../core/../external/compare_compat.hpp:26:10: error: no template named '__is_cpp17_random_access_iterator'
                                  && __is_cpp17_random_access_iterator<_InputIterator2>::value) {
                                     ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/utilities/filestat/filestat.cpp:58:65: warning: 'u8path<std::string>' is deprecated [-Wdeprecated-declarations]
        const auto filePath = dv::pathResolveExisting(std::filesystem::u8path(filestat["--file"]->as<std::string>()));
                                                                       ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.5.sdk/usr/include/c++/v1/__filesystem/u8path.h:83:27: note: 'u8path<std::string>' has been explicitly marked deprecated here
_LIBCPP_INLINE_VISIBILITY _LIBCPP_DEPRECATED_WITH_CHAR8_T
                          ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.5.sdk/usr/include/c++/v1/__config:1055:45: note: expanded from macro '_LIBCPP_DEPRECATED_WITH_CHAR8_T'
#    define _LIBCPP_DEPRECATED_WITH_CHAR8_T _LIBCPP_DEPRECATED
                                            ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.5.sdk/usr/include/c++/v1/__config:1010:49: note: expanded from macro '_LIBCPP_DEPRECATED'
#      define _LIBCPP_DEPRECATED __attribute__((__deprecated__))
                                                ^
1 warning and 2 errors generated.
[3/9] /opt/homebrew/bin/ccache ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DBOOST_NOWIDE_NO_LIB -DBOOST_STACKTRACE_GNU_SOURCE_NOT_REQUIRED -DCLI11_COMPILE -DHAVE_PTHREADS=1 -DOS_MACOS=1 -DOS_UNIX=1 -D_DARWIN_C_SOURCE=1 -D_DEFAULT_SOURCE=1 -D_FILE_OFFSET_BITS=64 -D_POSIX_C_SOURCE=200809 -D_XOPEN_SOURCE=700 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/eigen3 -fPIC -O3 -DNDEBUG -std=c++20 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.5.sdk -MD -MT utilities/imu-bias-estimation/CMakeFiles/dv-imu-bias-estimation.dir/imu-bias-estimation.cpp.o -MF utilities/imu-bias-estimation/CMakeFiles/dv-imu-bias-estimation.dir/imu-bias-estimation.cpp.o.d -o utilities/imu-bias-estimation/CMakeFiles/dv-imu-bias-estimation.dir/imu-bias-estimation.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/utilities/imu-bias-estimation/imu-bias-estimation.cpp
FAILED: utilities/imu-bias-estimation/CMakeFiles/dv-imu-bias-estimation.dir/imu-bias-estimation.cpp.o 
/opt/homebrew/bin/ccache ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DBOOST_NOWIDE_NO_LIB -DBOOST_STACKTRACE_GNU_SOURCE_NOT_REQUIRED -DCLI11_COMPILE -DHAVE_PTHREADS=1 -DOS_MACOS=1 -DOS_UNIX=1 -D_DARWIN_C_SOURCE=1 -D_DEFAULT_SOURCE=1 -D_FILE_OFFSET_BITS=64 -D_POSIX_C_SOURCE=200809 -D_XOPEN_SOURCE=700 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/eigen3 -fPIC -O3 -DNDEBUG -std=c++20 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.5.sdk -MD -MT utilities/imu-bias-estimation/CMakeFiles/dv-imu-bias-estimation.dir/imu-bias-estimation.cpp.o -MF utilities/imu-bias-estimation/CMakeFiles/dv-imu-bias-estimation.dir/imu-bias-estimation.cpp.o.d -o utilities/imu-bias-estimation/CMakeFiles/dv-imu-bias-estimation.dir/imu-bias-estimation.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/utilities/imu-bias-estimation/imu-bias-estimation.cpp
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/utilities/imu-bias-estimation/imu-bias-estimation.cpp:1:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/camera/calibration_set.hpp:3:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/camera/../core/utils.hpp:4:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/camera/../core/../external/compare_compat.hpp:25:16: error: no template named '__is_cpp17_random_access_iterator'
        if constexpr (__is_cpp17_random_access_iterator<_InputIterator1>::value
                      ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/camera/../core/../external/compare_compat.hpp:26:10: error: no template named '__is_cpp17_random_access_iterator'
                                  && __is_cpp17_random_access_iterator<_InputIterator2>::value) {
                                     ^
2 errors generated.
[4/9] /opt/homebrew/bin/ccache ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DBOOST_NOWIDE_NO_LIB -DBOOST_STACKTRACE_GNU_SOURCE_NOT_REQUIRED -DCLI11_COMPILE -DHAVE_PTHREADS=1 -DOS_MACOS=1 -DOS_UNIX=1 -D_DARWIN_C_SOURCE=1 -D_DEFAULT_SOURCE=1 -D_FILE_OFFSET_BITS=64 -D_POSIX_C_SOURCE=200809 -D_XOPEN_SOURCE=700 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/eigen3 -fPIC -O3 -DNDEBUG -std=c++20 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.5.sdk -MD -MT utilities/tcpstat/CMakeFiles/dv-tcpstat.dir/tcpstat.cpp.o -MF utilities/tcpstat/CMakeFiles/dv-tcpstat.dir/tcpstat.cpp.o.d -o utilities/tcpstat/CMakeFiles/dv-tcpstat.dir/tcpstat.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/utilities/tcpstat/tcpstat.cpp
FAILED: utilities/tcpstat/CMakeFiles/dv-tcpstat.dir/tcpstat.cpp.o 
/opt/homebrew/bin/ccache ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DBOOST_NOWIDE_NO_LIB -DBOOST_STACKTRACE_GNU_SOURCE_NOT_REQUIRED -DCLI11_COMPILE -DHAVE_PTHREADS=1 -DOS_MACOS=1 -DOS_UNIX=1 -D_DARWIN_C_SOURCE=1 -D_DEFAULT_SOURCE=1 -D_FILE_OFFSET_BITS=64 -D_POSIX_C_SOURCE=200809 -D_XOPEN_SOURCE=700 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/eigen3 -fPIC -O3 -DNDEBUG -std=c++20 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.5.sdk -MD -MT utilities/tcpstat/CMakeFiles/dv-tcpstat.dir/tcpstat.cpp.o -MF utilities/tcpstat/CMakeFiles/dv-tcpstat.dir/tcpstat.cpp.o.d -o utilities/tcpstat/CMakeFiles/dv-tcpstat.dir/tcpstat.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/utilities/tcpstat/tcpstat.cpp
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/utilities/tcpstat/tcpstat.cpp:4:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/io/network_reader.hpp:3:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/io/camera_input_base.hpp:3:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/io/../core/core.hpp:3:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/io/../core/../data/depth_event_base.hpp:8:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/io/../core/../data/cvector.hpp:3:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/io/../core/../data/../external/compare_compat.hpp:25:16: error: no template named '__is_cpp17_random_access_iterator'
        if constexpr (__is_cpp17_random_access_iterator<_InputIterator1>::value
                      ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/dv-processing/src/cbe2a4d037-da936aba53.clean/include/dv-processing/io/../core/../data/../external/compare_compat.hpp:26:10: error: no template named '__is_cpp17_random_access_iterator'
                                  && __is_cpp17_random_access_iterator<_InputIterator2>::value) {
                                     ^
2 errors generated.
ninja: build stopped: subcommand failed.
```
</details>

--- a/ports/dv-processing/portfile.cmake
+++ b/ports/dv-processing/portfile.cmake
@@ -1,3 +1,10 @@
+vcpkg_download_distfile(
+    XCODE_15_3_FIX_PATCH
+    URLS https://gitlab.com/inivation/homebrew-inivation/-/raw/master/Formula/dv-processing-xcode-15.3-fix.patch
+    FILENAME "xcode-15.3-fix.patch"
+    SHA512 cdbf77ee9619dbdad6e088d730d6d8dd6cf1cc12edc134fa8c905e538e94692c6b89e86ecb14766b133b87071459599e73ed61491db804bde0b9776307a6f157
+)
+
 vcpkg_from_gitlab(
     GITLAB_URL https://gitlab.com/inivation
     OUT_SOURCE_PATH SOURCE_PATH
@@ -5,6 +12,8 @@ vcpkg_from_gitlab(
     REF 9cd21eede0c38e079e462cdce8434fcbe2a4d037
     SHA512 fc5d0083166ff4708e6d540d437429784f9f62b7c3b7fb4631abc27ee0e6f46e60314f5fcf571c6141352571fef52a32c85a8160c951b5243910a02a281b0855
     HEAD_REF rel_1.7
+    PATCHES
+        "${XCODE_15_3_FIX_PATCH}"
 )
 
 vcpkg_from_gitlab(

--- a/ports/dv-processing/vcpkg.json
+++ b/ports/dv-processing/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "dv-processing",
   "version": "1.7.9",
-  "port-version": 1,
+  "port-version": 2,
   "description": "Generic algorithms for event cameras. (C++20 required.)",
   "homepage": "https://gitlab.com/inivation/dv/dv-processing",
   "license": "Apache-2.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2394,7 +2394,7 @@
     },
     "dv-processing": {
       "baseline": "1.7.9",
-      "port-version": 1
+      "port-version": 2
     },
     "dx": {
       "baseline": "1.0.1",

--- a/versions/d-/dv-processing.json
+++ b/versions/d-/dv-processing.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "2ef0b75d6a6ec6e9472d1bb3c227d13d0e40a2db",
+      "version": "1.7.9",
+      "port-version": 2
+    },
+    {
       "git-tree": "cd165ee3da5e52cb1a2a9bf22a87d04c2e100d4e",
       "version": "1.7.9",
       "port-version": 1

